### PR TITLE
Several fixes before implement tuple compression

### PR DESCRIPTION
**Pull request by EvgenyMekhanik**
This patch set includes several fixes:
- fix incorrect resource release in preformance test
- fixed incorrect print of msgpack field with MP_UNKNOWN_EXTENTION type
- add filters in test to remove dependence on tuple size

### CODE REVIEW

File: changelogs/unreleased/fix-incorrect-print-of-msgpack-field.md
Comment (locker) [inline changelogs/unreleased/fix-incorrect-print-of-msgpack-field.md:4]: Nit: the line length limit is 80 symbols. I'd understood if you used 72 symbols, but not 45? I notice that you often trim lines too short. Please don't.

Comment (locker) [inline changelogs/unreleased/fix-incorrect-print-of-msgpack-field.md:4]: A user reading this change log (change logs are for users) wouldn't understand what it's about. Are there any user-visible effects (like possible crash?) If so, please mention it and, even better, open a ticket for the reference. If there isn't, better drop this change log altogether.

**Review by locker — CHANGES_REQUESTED** [review]
Please see one nit pick about the change log. Plus, a few minor general comments:

1. Let's keep `NO_DOC/NO_CHANGELOG` reason consistent and meaningful. If it's a bug fix, use `NO_DOC=bug fix` (we usually don't document bug fixes). If it's a test fix, use `NO_DOC=test fix`. `NO_DOC=minor changes` doesn't make any sense. `NO_DOC=minor changes in test` is unnecessarily verbose.
2. Please always set a meaningful PR title and description so that a second reviewer can understand what this is about and how you've come to this.
3. Please don't set 'full-ci' label until an approval unless you specifically need to test all platforms, and you're afraid that your changes are dangerous and may break something. Or you can leave it to me - I'll set it before merging your PR.

File: changelogs/unreleased/fix-incorrect-print-of-msgpack-field.md
Comment (EvgenyMekhanik) [reply changelogs/unreleased/fix-incorrect-print-of-msgpack-field.md]: Remove this changelog because it this fix is not visible to user

**Review by locker — APPROVED** [review]

**Pull request merged**
